enclave/core/sgx/backtrace.c

```diff
@@ -106,10 +151,16 @@ char** oe_backtrace_symbols_impl(
     char** argv = NULL;
 
     if (!buffer || size < 0)
+    {
+        retval = OE_INVALID_PARAMETER;
         goto done;
+    }
```

Comment:
OE_RAISE(OE_INVALID_PARAMETER);
?   And same question for line 160-163.
I think the difference is it's shorter, and it also logs the result.

Reply:
Updated.